src/main/java/mate/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/drivers/all")

```

Comment:
let's keep URL mapping from common mistakes 
<img width="493" alt="image" src="https://user-images.githubusercontent.com/61326568/191194055-5d1593c4-ced9-4b9c-8f8b-1e1d04190048.png">
```suggestion
@WebServlet(urlPatterns = "/cars/drivers")

```